### PR TITLE
[Hotfix] Add missing resource

### PR DESCRIPTION
**Pull request by joelverhagen**
This PR fixes the currently broken transform-user-to-organization flow: https://github.com/NuGet/NuGetGallery/issues/8344

This flow was broken by my move to .SDK-based projects. When converting NuGetGallery.Core.csproj to SDK-based, I missed an embedded resource item.

I have verified this on DEV. I'd like to take this to INT and PROD today to unblock our users.

### CODE REVIEW

**Review by skofman1 — APPROVED** [review]

File: src/NuGetGallery.Core/NuGetGallery.Core.csproj [hunk 1]
```diff
@@ -34,6 +34,14 @@
     <Compile Remove="Services\CorePackageFileService.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Infrastructure\MigrateUserToOrganization.sql" />
```
Comment (cristinamanum) [inline src/NuGetGallery.Core/NuGetGallery.Core.csproj:38]: Why is this needed?

Comment (joelverhagen) [reply src/NuGetGallery.Core/NuGetGallery.Core.csproj:38]: This is added automatically by the IDE when changing the type to embedded.

By default, *.sql (and many other file types) are included as `<None>` items in SDK-based projects.

It's not strictly necessary AFAIK (`<None>` has no effect) but I figured it's best to follow the IDE defaults to avoid future unexpected .csproj changes.

**Review by cristinamanum — APPROVED** [review]

File: src/NuGetGallery.Core/NuGetGallery.Core.csproj (hunk 1 above)
Comment (cristinamanum) [reply src/NuGetGallery.Core/NuGetGallery.Core.csproj:38]: It sounds good
